src/App.js

```diff
@@ -5,9 +5,11 @@ import { collection, getDocs } from "firebase/firestore";
 import { db } from "./firebase-config";
 import Navbar from "./components/Navbar";
 import Login from "./components/Login";
+import LogOut from "./components/Login/LogOut";
```

Comment:
why O is capital in `LogOut`? and why not `Login` like LogIn?  Try to use same pattrens

Reply:
OK I'll will follow Login/Logout convention.